README.md

```diff
@@ -83,11 +83,12 @@ Making it conform to AutoJSONDeserializable would result make it support this JS
 
 # Features #
 
-  * primitive JSON types (String, Int, Double)
+  * Primitive JSON types (String, Int, Double).
   * ISO8601 formatted dates
```

Comment:
The final dot should be put on every line here or on none. If we put dots, let's not forget this little one here 😉